src/components/AppOverlay.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <div
+    class="fixed inset-0 before:content-{''} before:absolute before:inset-0 z-[99999] before:z-[99998] before:bg-black before:opacity-[0.75] h-[100vh] flex justify-center items-center backdrop-blur-[20px]"
```

Comment:
`h-[100vh]` = `h-screen`
`opacity-[0.75]` = `opacity-75`

Add the backdrop-blur-[20px] in tailwind config
```
theme: {
extend: {
backdropBlur: {
'xar-sm': 20px
}
}
}
```

And then use it like `backdrop-blur-xar-sm`
Use this for all the custom styles not available in tailwind.